apispec/core.py

```diff
@@ -19,16 +19,22 @@
 
 def clean_operations(operations):
     """Ensure that all parameters with "in" equal to "path" are also required
-    as required by the Swagger specification.
+    as required by the Swagger specification, as well as normalizing any
+    references to global parameters.
 
     See https://github.com/swagger-api/swagger-spec/blob/master/versions/2.0.md#parameterObject.
 
     :param dict operations: Dict mapping status codes to operations
     """
+    get_ref = lambda x: x if isinstance(x, dict) else {'$ref': '#/parameters/' + x}
+
     for operation in (operations or {}).values():
-        for parameter in operation.get('parameters', []):
-            if parameter['in'] == 'path':
-                parameter['required'] = True
+        if 'parameters' in operation:
```

Comment:
I changed it in order to pass the current test suite. The previous code inserted an empty list  for `operation['parameters']` if it did not exist.